lti_consumer/api.py

```diff
@@ -180,29 +190,65 @@ def get_lti_1p3_content_url(config_id=None, block=None, hint=""):
         b. Deep Linking is configured, but a single ltiResourceLink was selected.
     2. The Deep Linking content presentation URL if there's more than one
        Lti DL content in the database.
+
+    The launch_information parameter is a dictionary that contains data necessary and related to an LTI 1.3 launch.
```

Comment:
Once we agree on these, I'd like to put this comment in a more centralized place. Or, at least, one each API method that accepts this dictionary (e.g. `get_lti_1p3_launch_start_url` and `get_lti_1p3_launch_info`).